README.md

```diff
@@ -3,336 +3,93 @@
+
+### Trust Model
+#### Actors
+There are 3 main actors in the core protocol:
+- Protocol Admin
+- Option Writer
+- Option Holder
+
+The **Protocol Admin** is the address to which protocol fees are swept, and can update the Protocol Admin address. No other permissioned actions are possible.
+
+**Option Writers** can create new option types and write option lots for any valid ERC-20 asset pair (excluding rebasing, fee-on-transfer, and ERC-777 tokens). Sufficient approval must be granted for the Settlement Engine to take custody of the requisite amount of the underlying asset. Once an option lot expires, the writer can redeem their Claim NFT to their share of the underlying and/or exercise assets.
+
+**Option Holders** can acquire options that have been written. This is accomplished via a standard ERC-1155 transaction to transfer the desired amount of option contracts. When exercising an option, they must hold enough of the exercise asset, and similarly to when writing an option, they must have granted sufficient approval for the Settlement Engine on the ERC20 exercise asset.
+
+#### Assets
+Each asset pair for which an option is written is custodied by the Settlement Engine. When an option is written, TODO. When an option is exercised, TODO. When an option lot claim is redeemed, TODO. The core Settlement Engine is agnostic with regard to the buying or selling of options contracts. These are ERC-1155 semi-fungible tokens which can be transacted freely, from the writer to any party wishing to hold and potentially exercise the option before expiry. The Settlement Engine emits a standard ERC-1155 TransferSingle event when 1 or more options of a given type changes hands.
+
+#### Actions
+What can each actor do, when, with how much of each asset?
+
+- `ERC1155`
+  - `anyone` can
+    - check balances of any address
+    - check ownership of any token
+    - check transfer approvals
+    - check supported interfaces (ERC165 and ERC1155)
+    - render the URI of any token
+- `OptionSettlementEngine`
+  - `anyone` can
+    - view protocol fee (expressed in basis points)
+    - view Protocol Admin ("feeTo") address
+    - check fee balance of any ERC20 asset
+    - check whether any token ID is an Option or an Option Lot Claim
+    - check positions of the underlying and exercise assets of any token
+    - get Option info for any token ID
+    - get Option Lot Claim info for any token ID
+    - check amount of a given TODO
+    - sweep accrued fees for any ERC20 asset to the Protocol Admin ("feeTo") address
+    - create an Option Type which does not exist yet
+    - write a new option lot for a given Option Type
+  - `ERC1155 Option fungible token holders` (i.e., Option Writers) can
+    - transfer (up to their amount held) Option fungible tokens to another address
+    - write new options to a given Option Lot Claim which they hold, before the expiry timestamp
```

Comment:
Not necessarily. These are the fungible token holders, not the NFT claim holders. This point refers to adding more options to a given lot